src/content.rs

```diff
@@ -176,8 +242,22 @@ pub struct IndexCache {
 ///
 /// If show_unpublished is `true`, this will include pages that Bartholomew has determined are
 /// unpublished.
-pub fn all_pages_load(dir: PathBuf, show_unpublished: bool) -> anyhow::Result<IndexCache> {
-    let files = all_files(dir)?;
+pub fn pages_by_glob_load(
+    dir: PathBuf,
+    glob_pattern: String,
+    show_unpublished: bool,
+) -> anyhow::Result<IndexCache> {
+    let mut files: Vec<PathBuf> = Vec::new();
+    let full_pattern = format!("{}{}", dir.to_string_lossy(), glob_pattern);
+    let full_path = PathBuf::from(&full_pattern);
+    for entry in glob(full_path.to_str().unwrap_or_default())? {
+        match entry {
+            Ok(path) => files.push(path),
+            Err(e) => {
+                bail!("Failed to read file glob: {e}")
```

Comment:
Does something further up the stack tell us which glob (and which page) has the problem?